deploy.cmd

```diff
@@ -0,0 +1,29 @@
+
+@echo off
+
+"tools\nuget\nuget.exe" "install" "FAKE.Core" "-OutputDirectory" "tools" "-ExcludeVersion" "-version" "4.4.2"
+
+:Build
+cls
+
+SET TARGET="CreatePackages"
+SET GITOWNER="octokit"
```

Comment:
Should this be named `GITHUBOWNER` to indicate this is specific to the GitHub API (rather than the Git repository)?

And `octokit` is an organization, so this is just going to fail miserably with the default. Not sure what to put here, so let's leave it empty.